app/utils/data.py

```
@@ -0,0 +1,9 @@
+import json
+
+def read_users():
+    with open("data/users.json", "r", encoding="utf-8") as f:
```

Ruff (0.14.11)

Unnecessary mode argument

Remove mode argument

(UP015)